packages/cubejs-schema-compiler/src/adapter/BaseFilter.ts

```diff
@@ -90,6 +90,17 @@ export class BaseFilter extends BaseDimension {
     }
   }
 
+  /**
+   * BaseFilter inherits from BaseDimension while Filter may be measure-based !!
+   */
+  public override dateFieldType() {
+    if (this.measure) {
+      return this.measureDefinition().type; // There is no fieldType in measure, but it seems that it's enough
```

Comment:
> Added line #L98 was not covered by tests

Can we add a test case for this?

Reply:
It's showing as `uncovered` because for now coverage is being collected only for `unit` tests. But this is an `integration` test